homeassistant/components/sonos/media_player.py

```diff
@@ -530,6 +534,9 @@ async def async_seen(self, player):
         self._poll_timer = self.hass.helpers.event.async_track_time_interval(
             self.update, datetime.timedelta(seconds=SCAN_INTERVAL)
         )
+        self._battery_timer = self.hass.helpers.event.async_track_time_interval(
+            self.update_battery_info, datetime.timedelta(seconds=SCAN_INTERVAL)
```

Comment:
Do we need to poll the battery at the same frequency? It should change far less frequently.

Reply:
The battery level changes less frequently, true, but I picked SCAN_INTERVAL to update the UI relatively soon after the power_source changes.